<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Relief Parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="errLabel">
     <property name="styleSheet">
      <string notr="true">color: Red</string>
     </property>
     <property name="text">
      <string>Error</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_15">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Vertex Selection&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidgetReferences">
     <property name="toolTip">
      <string>- select an item to highlight it
- double-click on an item to see the fillets</string>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="batchSize">
      <number>80</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="extrusionParameters">
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Parameters&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Relief Size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Gui::QuantitySpinBox" name="relief">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string>Thickness of sheet material used by this object</string>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="unit" stdset="0">
          <string>mm</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="updateView">
     <property name="text">
      <string>Update view</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::QuantitySpinBox</class>
   <extends>QWidget</extends>
   <header>Gui/QuantitySpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
